src/main/java/io/jenkins/pluginhealth/scoring/service/PluginService.java

```diff
@@ -37,6 +39,24 @@ public PluginService(PluginRepository pluginRepository) {
         this.pluginRepository = pluginRepository;
     }
 
+    public void updateDatabase(Plugin plugin) {
+        Optional<Plugin> pluginFromDatabase = pluginRepository.findByName(plugin.getName());
+
+        if (pluginFromDatabase.isPresent()) {
+            if (pluginFromDatabase.get().getReleaseTimestamp() != plugin.getReleaseTimestamp()) {
+                pluginFromDatabase.get().setReleaseTimestamp(plugin.getReleaseTimestamp());
+                saveOrUpdate(pluginFromDatabase.get());
+            }
+            if (pluginFromDatabase.get().getScm() != null && !pluginFromDatabase.get().getScm().equals(plugin.getScm())) {
```

Comment:
Is this a suggested (or scalable) way to check for differences in values for a field? How can I make it look smarter and have less LOC? Can you please give some hints?

Reply:
I don't think we need to check the previous values here. The update-center will always have the most up-to-date values for SCM and releaseTimestamp.

Reply:
So that would mean we'd run `saveOrUpdate()` method for every plugin. Meaning, that no matter whether the field values of a plugin record are the latest or not we would still be running an update query on the plugin records stored in the DB.